### ISSUE TITLE
Option for Single GPU?

### ISSUE THREAD

**Issue by hoomexsun** [jaywalnut310/vits#204]
Hi, I want to train the model using a single GPU device that I have from my institute. What adjustments are required?

**Comment by phigrey** [comment]
Not really, you can run training directly, however, it would take quite a long time by only using one GPU, even it is A100.

**Comment by hoomexsun** [comment]
Can you give me a rough estimation??

I meant running with 1 gpu vs with more

Dataset would be LJ Speech
